kubernetes/samples/scripts/create-fmw-infrastructure-domain/wlst/create-domain-script.sh

```diff
@@ -22,4 +22,4 @@ wlst.sh -skipWLSModuleScanning \
         -managedServerPort ${CUSTOM_MANAGEDSERVER_PORT} \
         -prodMode ${CUSTOM_PRODUCTION_MODE} \
         -managedServerCount ${CUSTOM_MANAGED_SERVER_COUNT} \
-        -clusterName ${CUSTOM_CLUSTER_NAME}
\ No newline at end of file
+        -clusterName ${CUSTOM_CLUSTER_NAME}
```

Comment:
Why would you eliminate the last newline here?  Normally, it is best practice to end text and script files with newline.

Reply:
The original file doesn't end with a new line. I don't know why it shows a diff. I didn't change to that line.

Reply:
Anyway, I added a newline in the end to follow the best practice. Please review.